# This file was auto-generated by Rapiddocs from our API Definition.

from ....core.exceptions.rapiddocs_http_exception import RapiddocsHTTPException


class BadRequest(RapiddocsHTTPException):
    def __init__(self) -> None:
        super().__init__(status_code=400, name="BadRequest")
